cypress/plugins/index.js

```diff
@@ -18,20 +18,21 @@ module.exports = (on, config) => {
   // `config` is the resolved Cypress config
   on("task", {
     createRule: data =>
-      fetch(`${mockAPI}bounce_rules/`, {
-        method: "POST",
-        body: JSON.stringify(data),
-        headers: { "Content-Type": "application/json" },
-      })
-        .then(res => res.json())
-        .catch(err => err),
+      axios
+        .post(`${mockAPI}bounce_rules/`, data, {
+          headers: { "Content-Type": "application/json" },
+        })
+        .then(res => res.data)
+        .catch(() => false),
     deleteRule: ruleId =>
-      fetch(`${mockAPI}bounce_rules/${ruleId}`, { method: "DELETE" })
-        .then(res => res)
-        .catch(err => err),
+      axios
+        .delete(`${mockAPI}bounce_rules/${ruleId}`)
+        .then(res => res.data)
+        .catch(() => false),
```

Comment:
yep exactly since we for the most part won't use most values on success or error, we would return some truthy value in the success case (object, array, boolean true) and in the error case return false so when we chain the results of this we can determine overall success/error of the task. Thanks for changing this back to axios. I'm also happy you got to use fetch in some way haha. We opted to go with node-fetch on ours because we use native fetch API on the browser too and want to mimic that on the server-side, but you guys should stick with axios since you already have it installed